data-reporting-service/src/main/resources/db/changelog/db.odse.changelog-15.1.yaml

```diff
@@ -0,0 +1,134 @@
+databaseChangeLog:
+  - changeSet:
+      id: 000
```

Comment:
@sveselev Will "000" work for Id?

Reply:
@rseshadri99 Anything works there, `ID` is `nvarchar` in liquibase `databasechangelog`. We could even reference to our JIRA ticket numbers in the Id, if wanted.